.rubocop.yml

```diff
@@ -69,6 +69,7 @@ Metrics/MethodLength:
   Exclude:
   - 'db/migrate/*'
   - spec/**/*
+  - 'app/services/first_mfa_enabled_for_user.rb'
```

Comment:
Could we add this as a comment to the file, that way it stays in the code, moves around with it, and we remember to clean it up if the code changes